packages/cli/src/commands/codegen.ts

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable prefer-const */
+import { fixParameters } from "../lib/helpers/parameters";
+import { CodeGenerator } from "../lib/generators/CodeGenerator";
+
+import chalk from "chalk";
+import axios from "axios";
+import { GluegunToolbox } from "gluegun";
+
+const defaultTemplate = "web3api.gen";
+const defaultManifest = "web3api.yaml";
```

Comment:
So if `web3api.yaml` doesn't exist, then use `web3api.yml` file?

Reply:
Yes, we need to check for both since both extensions are used heavily: https://stackoverflow.com/questions/21059124/is-it-yaml-or-yml

https://github.com/search?q=.yml&type=code